crates/octez/src/async/protocol.rs

```diff
@@ -63,6 +60,22 @@ impl Protocol {
 #[folder = "$CARGO_MANIFEST_DIR/resources/protocol_parameters/"]
 pub struct ProtocolParameterFile;
 
+#[derive(Clone)]
+pub struct ProtocolParameter {
+    protocol: Protocol,
+    parameter_file: Arc<tempfile::NamedTempFile>,
```

Comment:
Because `ProtocolParameter` needs to be clonable. `ProtocolParameter` needs to be clonable because it will be part of `JstzdConfig` and that config needs to be clonable so that it can be held by different entities that need it.